spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/LibraryControllerHelperService.java

```diff
@@ -673,6 +674,12 @@ public JSONObject addLibraryQC(HttpSession session, JSONObject json) {
     return JSONUtils.SimpleJSONError("Cannot add LibraryQC");
   }
 
+  private double round(double value, int numberOfDigitsAfterDecimalPoint) {
```

Comment:
Ick. It saddens me that Java doesn't have a rounding method. Since this is going to be converted to a string, it would be reasonable to use `DecimalFormat`, but meh.